src/lib/service/AwardCoService.js

```diff
@@ -24,7 +24,7 @@ class AwardCoService {
     let note = this.defaultNote;
     if (event.reason) {
       const buff = new Buffer.from(event.reason, 'base64');
-      note = buff.toString('UTF-8');
+      note = buff.toString('UTF-8') + " (via ${this.robot.name})";
```

Comment:
I _think_ this is what you want
```suggestion
      note = `${buff.toString('UTF-8')} (via ${this.robot.name})`;
```